utils/driverless.c

```diff
@@ -626,13 +540,85 @@ list_printers (int mode, int reg_type_no, int isFax)
     if (debug)
       fprintf(stderr, "DEBUG: Started %s (PID %d)\n", ippfind_argv[0],
 	      ippfind_ipp_pid);
-    listPrintersInArray(post_proc_pipe_ipp, service_uri_list_ipps, 0, mode,
-			isFax);
   }
+  /*
+  Reading the ippfind output in CUPS Array
+  */
+  if(reg_type_no >=1)
+  {
+    dup2(post_proc_pipe_ipps[0], 0);
+    close(post_proc_pipe_ipps[0]);
+    close(post_proc_pipe_ipps[1]);
 
-  close(post_proc_pipe_ipp[0]);
-  close(post_proc_pipe_ipp[1]);
+    fp = cupsFileStdin();
 
+    while ((bytes = cupsFileGetLine(fp, buffer, sizeof(buffer))) > 0) 
+    {
+      ippfind_output = (char *)malloc(8192*(sizeof(char)));
```

Comment:
@nidhijainpnp Hi Nidhi,

I don't  see any freeing this memory in the loop, would you mind fixing it?
Magic number '8192' doesn't give much info to a person reading it - would it be possible to create a ```#define <readable_name> 8192``` to help people understand why such number is used?